scala3/scaladoc.md

```diff
@@ -43,16 +43,18 @@ Furthermore, Scaladoc provides an easy way to configure your [social media links
 
 The following features are currently (May 2021) not stable to be released with scaladoc, however we are happy to hear your feedback. Each feature has its own thread at scala-lang contributors site, where you can share your opinions.
 
-### Snippets compiler
+### Snippet checking
```

Comment:
“checking” is a bit vague to me, I think “compiling” has a more intuitive meaning to me. But YMMV. Or maybe “type-checking”?

Reply:
changed